packages/block-library/src/cover/style.scss

```diff
@@ -130,7 +134,9 @@
 	h5,
 	h6,
 	.wp-block-subhead {
-		color: inherit;
+		&:not(.has-text-color) {
```

Comment:
Not really important, but shouldn't this just be `.wp-block-subhead:not(.has-text-color)`?

Reply:
I don't think so we also want to have selector p:not(.has-text-color) and h1:not(.has-text-color) etc... Having the selectors nested in this way ensures we apply the rule to all the selectors (not just .wp-block-subhead) that don't have a text color.

Reply:
Oh, whoops. I misread the SCSS. :stuck_out_tongue: